src/common/mfu_daos.h

```diff
@@ -47,6 +47,8 @@ typedef struct {
     bool allow_exist_dst_cont;          /* whether to allow the dst container to exist for DAOS API */
     enum daos_cont_props src_cont_type; /* type of the source container */
     enum daos_cont_props dst_cont_type; /* type of the destination container */
+    bool daos_preserve;                 /* preserve daos cont props and user attrs */
+    char *daos_preserve_path;           /* set path to write daos props and user attrs */
```

Comment:
@dsikich I wonder if we could simplify this by removing `daos_preserve` and using `daos_preserve_path != NULL` to mean `true`

Reply:
@daltonbohning yeah that would remove the extra variable.

Reply:
@daltonbohning this doesn't totally work actually, because I need to pass different values to daos connect depending on if DAOS is the source or destination.

Reply:
@dsikich Ah, that makes sense!